x-pack/solutions/security/packages/navigation/src/constants.ts

```diff
@@ -16,91 +16,10 @@ export enum LinkCategoryType {
-  mlDataVisualizer = 'ml:dataVisualizer',
-  mlNotifications = 'ml:notifications',
-  mlMemoryUsage = 'ml:memoryUsage',
-  mlAnomalyDetection = 'ml:anomalyDetection',
-  mlAnomalyExplorer = 'ml:anomalyExplorer',
-  mlSingleMetricViewer = 'ml:singleMetricViewer',
-  mlSuppliedConfigurations = 'ml:suppliedConfigurations',
-  mlSettings = 'ml:settings',
-  mlDataFrameAnalytics = 'ml:dataFrameAnalytics',
-  mlResultExplorer = 'ml:resultExplorer',
-  mlAnalyticsMap = 'ml:analyticsMap',
-  mlNodesOverview = 'ml:nodesOverview',
-  mlNodes = 'ml:nodes',
-  mlFileUpload = 'ml:fileUpload',
-  mlIndexDataVisualizer = 'ml:indexDataVisualizer',
-  mlESQLdataVisualizer = 'ml:esqlDataVisualizer',
-  mlDataDrift = 'ml:dataDrift',
-  mlExplainLogRateSpikes = 'ml:logRateAnalysis',
-  mlLogPatternAnalysis = 'ml:logPatternAnalysis',
-  mlChangePointDetections = 'ml:changePointDetections',
-  // Dev Tools
-  // Ref: src/platform/packages/private/default-nav/devtools/default_navigation.ts
-  devTools = 'dev_tools:',
-  // Fleet
-  // Ref: x-pack/platform/plugins/shared/fleet/public/deep_links.ts
-  fleet = 'fleet:',
-  fleetAgents = 'fleet:agents',
-  fleetPolicies = 'fleet:policies',
-  fleetEnrollmentTokens = 'fleet:enrollment_tokens',
-  fleetUninstallTokens = 'fleet:uninstall_tokens',
-  fleetDataStreams = 'fleet:data_streams',
-  fleetSettings = 'fleet:settings',
-  // Integrations
-  // No deepLinkId registered, using path for the security search
-  integrationsSecurity = 'integrations:/browse/security',
-  // Management
-  // Ref: src/platform/packages/private/default-nav/management/default_navigation.ts
-  management = 'management:',
-  managementIngestPipelines = 'management:ingest_pipelines',
-  managementPipelines = 'management:pipelines',
-  managementIndexManagement = 'management:index_management',
-  managementTransforms = 'management:transform',
-  managementMaintenanceWindows = 'management:maintenanceWindows',
-  managementTriggersActions = 'management:triggersActions',
-  managementCases = 'management:cases',
-  managementTriggersActionsConnectors = 'management:triggersActionsConnectors',
-  managementReporting = 'management:reporting',
-  managementJobsListLink = 'management:jobsListLink',
-  managementDataViews = 'management:dataViews',
-  managementObjects = 'management:objects',
-  managementApiKeys = 'management:api_keys',
-  managementTags = 'management:tags',
-  managementFiles = 'management:filesManagement',
-  managementSpaces = 'management:spaces',
-  managementSettings = 'management:settings',
-  managementMonitoring = 'monitoring:',
-  // Cloud UI
-  // These are links to Cloud UI outside Kibana
-  // Special Format: <cloud>:<cloudUrlKey>
-  // cloudUrlKey Ref: x-pack/solutions/security/plugins/security_solution_serverless/public/navigation/links/util.ts
-  cloudUsersAndRoles = 'cloud:usersAndRoles',
-  cloudBilling = 'cloud:billing',
-  cloudPerformance = 'cloud:performance',
+export enum SecurityGroupName {
```

Comment:
Can you add a comment similar to the comment that was here before for `ExternalPageName`?

Reply:
done! https://github.com/elastic/kibana/pull/217890/commits/0bfd1a68a2ccd847550743112129ea29210fe077